script/tool/lib/src/common/package_looping_command.dart

```diff
@@ -401,4 +416,21 @@ abstract class PackageLoopingCommand extends PluginCommand {
     }
     _printError(failureListFooter);
   }
+
+  /// Prints [message] in [color] unless [captureOutput] is set, in which case
+  /// it is printed without color.
+  void _printColorized(String message, Styles color) {
+    if (captureOutput) {
+      print(message);
+    } else {
+      print(Colorize(message)..apply(color));
+    }
+  }
+
+  /// Returns a duration [d] formatted as minutes:seconds. Does not use hours,
+  /// since time logging is primarily intended for CI, where durations should
+  /// always be less than an hour.
+  String _formatDuration(Duration d) {
+    return '${d.inMinutes}:${(d.inSeconds % 60).toString().padLeft(2, '0')}';
```

Comment:
The iso standard for durations would be `mMsS`  Having a colon usually denotes a time.

https://en.wikipedia.org/wiki/ISO_8601#Durations

Reply:
TIL there's an ISO standard duration format. It's hard to read though, and this is for human consumption; I'll change the elapsed time per package to "Xm YYs" so it's easy to read and not ambiguous. (That's also what Cirrus uses).

The relative start time (shown in the header) is effectively a time, so that should stay with this format. (Also matching Cirrus)